lib/findingaids/ead/behaviors.rb

```diff
@@ -114,6 +125,10 @@ def ead_language_fields fields = Hash.new
     return fields
   end
 
+  def fix_subfield_demarcators(value)
+    v = value.gsub(/\|\w{1}/,"--")
+    return v
+  end
```

Comment:
@eshadatta And you can add a unit test for that in `lib/findingaids/ead/behaviors_spec.rb`